ls1intum/Artemis
# [Exam] Include last commit hash in exam summary

### Checklist

### Motivation and Context



The student should see the last commit hash in the exam summary

### Description


- Add last commit hash to exam summary

### Steps for Testing


1. Log in to Artemis
2. Participate in an exam with programming exercises
3. See the 'last commit' field in the exam summary

**Notes:** Right now it can happen, that the last commit hash is not directly available, but only after a page refresh. This is a known issue and being worked on.

<img width="723" alt="Bildschirmfoto 2020-07-01 um 18 13 05" src="https://user-images.githubusercontent.com/13920539/86267115-8525ff00-bbc6-11ea-9a94-d73e5b60f203.png">
